### PR TITLE
Added a command to redstack to run the server in fake mode.

### PR DESCRIPTION
**Pull request by TimSimpsonR**

### CODE REVIEW

**Comment by hub-cap** [comment]
+1

**Comment by cp16net** [comment]
+1 FAKER

File: scripts/redstack [hunk 1]
```diff
@@ -725,7 +725,22 @@ EOF
...
+        --config-file=/tmp/reddwarf.conf.sample $@
+}
+
+function cmd_run_fake() {
+    CONF_FILE=$PATH_REDDWARF/etc/reddwarf/reddwarf.conf.test
+    cd $PATH_REDDWARF
+    set -e
+    rm reddwarf_test.sqlite
+    set +e
+    bin/reddwarf-manage --config-file=$CONF_FILE db_sync /
```
Comment (ed-) [inline scripts/redstack]: You keep using that `/`. I do not think it means what you think it means.

**Comment by rnirmal** [comment]
+1

**Comment by ed-** [comment]
+1

**Pull request merged**